.circleci/config.yml

```diff
@@ -10,7 +10,6 @@ jobs:
      resource_class: xlarge
      working_directory: /source
      environment:
-       BAZEL_REMOTE_CACHE: https://storage.googleapis.com/envoy-circleci-bazel-cache/
```

Comment:
I don't think this was related to the failures you saw.  Let's make sure we revert this before merge.

Reply:
agreed. but it won't pass CI without it, afaict

Reply:
Looks like CI is starting to work again without removing the remote cache.